### ISSUE TITLE
Incorrect validation for clock_time_get

### ISSUE THREAD

**Issue by saghul** [wasmerio/wapm-cli#141]
### Describe the bug

According to the readme (and what I experience) `clock_time_get` has the following interface:

```
  (func (import "wasi_unstable" "clock_time_get") (param i32 i32 i32) (result i32))
```

But according to https://github.com/WebAssembly/WASI/blob/b81f586ecdca184764e91587a18d59b1d85242c9/design/WASI-core.md#__wasi_clock_time_get I think it should be:

```
  (func (import "wasi_unstable" "clock_time_get") (param i32 i64) (result i32))
```

I'ma bit of a nood here, so I may be totally off, but I see 2 discrepancies: the parameter count is 2 not 3 and the second one (__wasi_timestamp_t) should be u64: https://github.com/WebAssembly/WASI/blob/b81f586ecdca184764e91587a18d59b1d85242c9/design/WASI-core.md#__wasi_timestamp_t-uint64_t

This is what happens when I try to publish my package QuickJS (https://github.com/saghul/wasi-lab/tree/master/qjs-wasi)

```
Error: WASM file "/Users/saghul/src/wasi-lab/qjs-wasi/build/qjs.wasm" detected as invalid because InterfaceViolated { errors: ["Type mismatch in params in imported func \"wasi_unstable\" \"clock_time_get\": argument 2, expected i32 found i64"] }
```

Copy and paste the result of executing the following in your shell, so we can know the version of `wapm-cli`, Rust (if available) and architecture of your environment.

```bash
wapm-cli 0.3.5
```

**Comment by saghul** [comment]
Ping @syrusakbary :-)

**Comment by saghul** [comment]
Bump @syrusakbary I'm trying to update the `duktape` package but I still run into this error with wapm-cli 0.3.7 and wasmer 0.7.0. How did you publish the first version?

**Comment by saghul** [comment]
@MarkMcCaskey Hey, maybe you can help with this?

**Comment by MarkMcCaskey** [comment]
Hi!  Sorry about the delay, I think this is a caching issue.  I believe we've fixed this on the server, but we don't currently clear the cache of our interfaces.  This is something that needs to change in wapm but in the meantime you can fix this on your end with sqlite!

in your `WASMER_DIR` (`$HOME/.wasmer` by default), you should have a `wapm.sqlite` file.

run
```sh
wapm install -g sqlite
wapm run sqlite --dir=$WASMER_DIR $WASMER_DIR/wapm.sqlite
```

Once in the sqlite repl, enter `delete from wasm_interfaces;` to clear the cache.  Exit the repl and run `wapm validate` again and it should be fixed!.

Oh, and I found another bug while writing this up 🤦‍♂ , it's not possible to run `wapm run` in the `$WASMER_DIR` because of the global config file called `wapm.toml` which is in a different format than normal `wapm.toml`s.

And the solution to this problem in general is probably to clear the cache on a fixed timer so bugs like this don't happen again.

Let me know if that doesn't work you and I'll prioritize fixing it

**Comment by saghul** [comment]
🎉🎉🎉

```
$ wapm publish
Successfully published package `duktape@0.0.2`
$ wapm publish
Successfully published package `quickjs@0.0.2`
``` 

Thanks a lot!
